### ISSUE TITLE
Mak/Nurse TOR phosphorylation data

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#3141]
- [x] I added a file to svn repository PMID_34296454_modifications.tsv for Tiffs TOR phosphoproteome
I think I managed to convert everything OK, but when I exported from the spreadsheet I got some weirdness that I cannot  seem to get rid of with a perl one-liner

The PMID has <CA> inserted:
<img width="470" alt="Screenshot 2021-10-28 at 17 39 49" src="https://user-images.githubusercontent.com/7359272/139299027-5be3bd2a-561f-462a-aa4d-7f43967249ef.png">

can either of you figure out how to get rid of this ? @kimrutherford @mah11 ?
There is no hurry because I would like it to be loaded AFTER I added the final term to the curation session and approved it, otherwise I won't be able to open the session... (Is there is a way to flag a session so that it isn't loaded?)

- [ ] the multiplicity score is still present in column 10, so it will need to be removed, or edited (unless additional columns are ignored, in which case it can just sit there. I have asked Tiffany for more information about this.

**Comment by mah11** [comment]
Based on additional info from Tiffany, I think we can include the "multiplicity" numbers as another extension type.

If I understand correctly, these annotations will get loaded once someone adds the new extension relations to the list [here](https://github.com/pombase/pombase-legacy/blob/master/load-pombase-chado.yaml#L233). For now, I've edited the file to convert multiplicity entries to extensions, and added `multiplicity` to the relations mini-ontology (at pombe-embl/mini-ontologies/PSI-MOD_extension_relations.obo). I can commit my file edits and do the load-yaml file edit whenever you're ready.

**Comment by ValWood** [comment]
I'd like to get the missing term into the spreadsheet first so that all of the modifications don't appear in Canto (or does this have no bearing on that happening?), if not, go ahead. I'm only waiting on one FYPO term to finish the small-scale part of the session.

**Comment by mah11** [comment]
Um, do you mean you want to get the missing term into the *Canto session*? Anyway, I'll do FYPO stuff today, and the modification stuff tomorrow.

**Comment by ValWood** [comment]
I'm waiting on a term for the Canto session. I wanted to get the Cant session finished  before the humungous dataset is loaded as it makes the Canto session difficult to load (in my experience). But it doesn't really matter, either order.

**Comment by mah11** [comment]
Yes, I figured that out despite "spreadsheet" in your earlier comment:

> I'd like to get the missing term into the spreadsheet first

**Comment by mah11** [comment]
OK, I did the data file and yaml file edits. I think that means this can close if the data look good tomorrow.

**Comment by mah11** [comment]
needs loader enhancement per https://github.com/pombase/pombase-chado/issues/880

**Comment by ValWood** [comment]
Can we close this since there is another ticket for thee multiplicity?

**Comment by mah11** [comment]
> Can we close this ...?

I think so.
